src/instructlab/utils.py

```diff
@@ -541,3 +541,20 @@ def split_hostport(hostport: str) -> tuple[str, int]:
     if not hostname or not port:
         raise ValueError(f"Invalid host-port string: '{hostport}'")
     return hostname, port
+
+
+def is_oci_repo(repo_url: str) -> bool:
```

Comment:
Given that check for a single string, I'd remove this function and do the check with an `if` statement.

Reply:
I just made this a function because our oci check is very simple right now, but i thought we may need to make this more robust in the future, so we could develop this logic later and it would be more testable this way